### PR TITLE
mocktikv:change the name of some structs

### PR DESCRIPTION
**Pull request by zhexuany**
## What have you changed? (mandatory)

I just change some name of the struct and add more comment which improve the readability of the code. 

## How has this PR been tested? (mandatory)
It does not need any test.

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)
No

## Does this PR affect tidb-ansible update? (mandatory)
No.

## Does this PR need to be added to the release notes? (mandatory)
No.

### CODE REVIEW

File: store/mockstore/mocktikv/analyze.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ import (
 	"golang.org/x/net/context"
 )
 
-func (h *rpcHandler) handleCopAnalyzeRequest(req *coprocessor.Request) *coprocessor.Response {
+func (h *mockRPCHandler) handleCopAnalyzeRequest(req *coprocessor.Request) *coprocessor.Response {
```
Comment (zz-jason) [inline store/mockstore/mocktikv/analyze.go]: I think the old name is better. It's already in the "mocktikv" package, everything can be considered "mock"

Comment (tiancaiamao) [reply store/mockstore/mocktikv/analyze.go]: Agree.

Comment (coocood) [reply store/mockstore/mocktikv/analyze.go]: +1

**Comment by shenli** [comment]
@zhexuany Please address the comment.

**Comment by XuHuaiyu** [comment]
@zhexuany friendly ping, comments not been addressed for several days.

**Comment by coocood** [comment]
LGTM

**Comment by coocood** [comment]
@zz-jason PTAL

File: store/mockstore/mocktikv/mock.go [hunk 2]
```diff
@@ -18,8 +18,8 @@ import (
 	"github.com/pingcap/pd/pd-client"
 )
 
-// NewTestClient creates a TiKV client and PD client from options.
-func NewTestClient(cluster *Cluster, mvccStore MVCCStore, path string) (*RPCClient, pd.Client, error) {
+// NewPDAndTiKVClient creates a TiKV client and PD client from options.
+func NewPDAndTiKVClient(cluster *Cluster, mvccStore MVCCStore, path string) (*RPCClient, pd.Client, error) {
```
Comment (shenli) [inline store/mockstore/mocktikv/mock.go]: NewTiKVAndPDClient is better. Because it has the same order with the return values.

Comment (zhexuany) [reply store/mockstore/mocktikv/mock.go]: Gotcha. Working on it right away.

**Pull request merged**